models/sipnet/R/met2model.SIPNET.R

```diff
@@ -147,12 +147,15 @@ met2model.SIPNET <- function(in.path, in.prefix, outfolder, start_date, end_date
       }
       
       Rain <- ncdf4::ncvar_get(nc, "precipitation_flux")
-      pres <- ncdf4::ncvar_get(nc,'air_pressure') ## in pascal
+      
+      press <- ncdf4::ncvar_get(nc,'air_pressure') ## in pascal
+
       SW <- ncdf4::ncvar_get(nc, "surface_downwelling_shortwave_flux_in_air")  ## in W/m2
       
       PAR <- try(ncdf4::ncvar_get(nc, "surface_downwelling_photosynthetic_photon_flux_in_air"))  ## in umol/m2/s
       if (!is.numeric(PAR)) {
-        PAR <- PEcAn.utils::ud_convert(PEcAn.data.atmosphere::sw2ppfd(SW), "umol ", "mol")
+        #PAR <- PEcAn.utils::ud_convert(PEcAn.data.atmosphere::sw2ppfd(SW), "umol ", "mol")
```

Comment:
is this still needed?

Reply:
good catch this is left over from debugging